pettingzoo/atari/base_atari_env.py

```diff
@@ -9,7 +9,7 @@
 
 from pettingzoo.utils import wrappers
 from pettingzoo.utils.conversions import parallel_to_aec_wrapper, parallel_wrapper_fn
-from pettingzoo.utils.env import ParallelEnv
+from pettingzoo import ParallelEnv
```

Comment:
This is good, we want people to know they can just do from pettingzoo import ParallelEnv